wiki/development/coding-guidelines.md

```diff
@@ -61,42 +59,40 @@ class ACE_Settings {
+- There will not be any empty stringtable language values.
+- All stringtables will follow the format as specified by [Tabler](https://github.com/bux578/tabler) and the [translation guidelines](http://ace3mod.com/wiki/development/how-to-translate-ace3.html) form.
 
 
 ## 2. Macro Usage
 
 ### 2.1. Module/PBO specific Macro Usage
 The family of `GVAR` macro's define global variable strings or constants for use within a module. Please use these to make sure we follow naming conventions across all modules and also prevent duplicate/overwriting between variables in different modules. The macro family expands as follows, for the example of the module 'balls':
 
-
 | Macros |  Expands to |
 | -------|---------|
-|`GVAR(face)` | ace_balls_face |
-|`QGVAR(face)` | "ace_balls_face" |
-|`EGVAR(balls,face)` | ace_balls_face |
-|`EGVAR(leg,face)` | ace_leg_face |
-|`QEGVAR(leg,face)` | "ace_leg_face" |
+|`GVAR(face)` | `ace_balls_face` |
+|`QGVAR(face)` | `"ace_balls_face"` |
+|`EGVAR(balls,face)` | `ace_balls_face` |
+|`EGVAR(leg,face)` | `ace_leg_face` |
+|`QEGVAR(leg,face)` | `"ace_leg_face"` |
 
-There also exists the FUNC family of Macros:
+There also exists the `FUNC` family of Macros:
 
 | Macros  |  Expands to |
 | -------|---------|
-|`FUNC(face)` | ace_balls_fnc_face or the call trace wrapper for that function.|
-|`EFUNC(balls,face)` | ace_balls_fnc_face or the call trace wrapper for that function.|
-|`EFUNC(leg,face) `| ace_leg_fnc_face or the call trace wrapper for that function.|
-|`DFUNC(face)` | ace_balls_fnc_face and will ALWAYS be the function global variable.|
-|`DEFUNC(leg,face)` | ace_leg_fnc_face and will ALWAYS be the function global variable.|
-|`QFUNC(face)` | "ace_balls_fnc_face" |
-|`QEFUNC(leg,face)` | "ace_leg_fnc_face" |
+|`FUNC(face)` | `ace_balls_fnc_face` or the call trace wrapper for that function. |
+|`EFUNC(balls,face)` | `ace_balls_fnc_face` or the call trace wrapper for that function. |
+|`EFUNC(leg,face) `| `ace_leg_fnc_face` or the call trace wrapper for that function. |
+|`DFUNC(face)` | `ace_balls_fnc_face` and will ALWAYS be the function global variable. |
+|`DEFUNC(leg,face)` | `ace_leg_fnc_face` and will ALWAYS be the function global variable. |
+|`QFUNC(face)` | `"ace_balls_fnc_face"` |
+|`QEFUNC(leg,face)` | `"ace_leg_fnc_face"` |
 
 The `FUNC` and `EFUNC` macros should NOT be used inside `QUOTE` macros if the intention is to get the function name or assumed to be the function variable due to call tracing (see below). If you need to 100% always be sure that you are getting the function name or variable use the `DFUNC` or `DEFUNC` macros. For example `QUOTE(FUNC(face)) == "ace_balls_fnc_face"` would be an illegal use of `FUNC` inside `QUOTE`.
 
 Using `FUNC` or `EFUNC` inside a `QUOTE` macro is fine if the intention is for it to be executed as a function.
 
-#### 2.1.1. FUNC Macros, Call Tracing, and Non-ACE3 /Anonymous Functions
-
+#### 2.1.1. `FUNC` Macros, Call Tracing, and Non-ACE3/Anonymous Functions
```

Comment:
I'd avoid using backticks in headers (don't think that is properly styled in css)